Procfile

```diff
@@ -1,2 +1,2 @@
-web:        babel-node bin/main.js server
-scheduler:  babel-node bin/main.js scheduler
\ No newline at end of file
+web:        babel-node src/main.js server
```

Comment:
so this might work, but it will end up actually not using the compiled (and tested) javascript code.  Instead, what you should be doing is `node lib/main.js ...` for both entries.